ldj-spring-renderer-example/src/main/java/com/infosupport/ldoc/springexample/ampq/QueueInteraction.java

```diff
@@ -0,0 +1,56 @@
+package com.infosupport.ldoc.springexample.ampq;
+
+import java.util.List;
+
+public record QueueInteraction(
+    // The actor (Java classes in this case) that either reads or posts a message in a queue
+    String actor,
+    // Whether actor reads or posts a message from/to a queue
+    QueueInteractionKind kind,
+    // The queue a message is either read/posted from/on
+    String queue,
+    // Class name of the POJO that is the message
+    //  NOTE: In order to support message type matching across multiple applications, in this
+    //    sample, the plain class name is used (as opposed to the fqdn)
+    String messageType,
+
+    // TODO note: Hacky: assumes this is Read, and contains solely post interactions that happen
```

Comment:
This highly depends on the final methodology of the sample renderers. So consider this as a note in case we want to have a model similar to this approach.